src/components/Header/styles/title.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+const Title = styled.div`
+  margin-top: ${({ theme }) => theme.unit.extraSmall}px;
+
+  &.withDivider {
+    margin-bottom: 8px;
+    padding-bottom: 8px;
+    border-bottom: 2px solid;
+    border-bottom-color: ${({ dividerColor }) => dividerColor || '#333'};
```

Comment:
What do you think about using the theme variables for the `border-bottom-color`, `margin-bottom` and `padding-bottom` properties?